jdbc-handler/src/main/java/org/apache/hive/storage/jdbc/dao/TeradataDatabaseAccessor.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hive.storage.jdbc.dao;
+
+/**
+ * Teradata specific data accessor. This is needed because Teradata JDBC drivers do not support generic LIMIT and OFFSET
+ * escape functions
+ */
+public class TeradataDatabaseAccessor extends GenericJdbcDatabaseAccessor {
+
+  @Override
```

Comment:
nit: this file seems unrelated to the problem, should we create another jira for the feature? I think it makes better for porting or tracking the issue.

Reply:
Not a nit. There should only be one file in this change, not sure how others got pulled in. Let me fix that.

Reply:
Fixed.